requirements.txt

```diff
@@ -7,4 +7,5 @@ django-cors-headers==2.2.0
 Faker==0.8.13
 factory-boy==2.10.0
 imailing==1.0.0
-django-import-export==1.0.0
\ No newline at end of file
+django-import-export==1.0.0
+icalendar
```

Comment:
Here is a missing newline, be sure to add it.

Reply:
Yep, and we will need a specific version too! :p